lib/govuk_index/presenters/indexable_content_presenter.rb

```diff
@@ -15,9 +22,10 @@ def indexable_content
     attr_reader :details, :sanitiser
 
     def elements
-      details.flat_map do |key, value|
+      details.flat_map { |key, value|
```

Comment:
I am sure rubocop normally complains about multiline blocks with {} syntax.... I agree with this rule.